db/migrate/20150909055225_add_include_conversation_in_replies_to_tenants.rb

```diff
@@ -0,0 +1,18 @@
+class AddIncludeConversationInRepliesToTenants < ActiveRecord::Migration
+  def change
+    if Tenant.postgresql?
+      old = Tenant.connection.schema_search_path
+      Tenant.connection.schema_search_path = 'public'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.